src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1554,8 +1554,11 @@ class ApplicationManagerImpl
   uint32_t navi_close_app_timeout_;
   uint32_t navi_end_stream_timeout_;
 
-  std::vector<TimerSPtr> timer_pool_;
-  sync_primitives::Lock timer_pool_lock_;
+  std::vector<TimerSPtr> close_app_timer_pool_;
+  std::vector<TimerSPtr> end_stream_timer_pool_;
```

Comment:
@mkorniichuk why did you divided the timers pool?

Reply:
@AKalinich-Luxoft please, take a look at https://github.com/smartdevicelink/sdl_core/pull/3202#discussion_r365530445